ouroboros/templates/emails/application/approved.html

```diff
@@ -23,7 +23,7 @@ <h2 style="margin-bottom: 20px; font-size: 16px; color: #777777">Congratulations
 				font-weight: normal;
 				color: #777777;
 				margin-bottom: 20px;">
-			Please <span style="color: #ED768B; font-weight: 700">RSVP at the link below by Friday, January 11th</span> to confirm your attendance. If you do not RSVP by then, you risk giving up your spot.
+			Please <span style="color: #ED768B; font-weight: 700">RSVP at the link below by Friday, August 30th</span> to confirm your attendance.
```

Comment:
The RSVP deadline for each hacker is [`DAYS_TO_RSVP`](https://github.com/tamuhack-org/Ouroboros/blob/2655e909fe07cc85cc0dbb7e95a233442f1dfd19/ouroboros/ouroboros/settings/customization.py#L6) days after the email is received, which makes this statement misleading